docs/docs-beta/docs/dagster-plus/deployment/hybrid/agents/local.md

```diff
@@ -1,8 +1,55 @@
 ---
-title: "Local agents"
+title: "Running a Dagster+ agent locally"
 displayed_sidebar: "dagsterPlus"
 sidebar_position: 40
 sidebar_label: "Local"
 ---
 
-# Local agents
\ No newline at end of file
+# Running a Dagster+ agent locally
+
+This page provides instructions for running the [Dagster+ agent](dagster-plus/getting-started/whats-dagster-plus#Agents) locally. Local agents are a good way to experiment with Dagster+ locally before deploying a more scalable hybrid agent (like [Kubernetes](dagster-plus/hybrid/agents/kubernetes) or [Amazon ECS](dagster-plus/hybrid/agents/amazon-ecs-new-vpc).
+
+## Installation
+
+
+### Step 1: Install dagster-cloud
+
+```shell
+pip install dagster-cloud
+```
+
+## Step 2: Configure the agent
+
+[Generate an agent token](dagster-plus/deployment/tokens) and configure your agent
+
+```shell
+dagster-cloud configure
```

Comment:
Does this do what we want here? My understanding was that `dagster-cloud configure` has a preference for setting up the user's token interactively... but I think in this case they should be manually configuring the API token via `dagster.yaml`?

Reply:
Oh yeah you're right. That's what I get for writing this guide from memory instead of actually dogfooding it 😅